amazon-kinesis-client/src/test/java/software/amazon/kinesis/retrieval/fanout/FanOutConsumerRegistrationTest.java

```diff
@@ -188,11 +188,10 @@ public void testNewRegisterStreamConsumer() throws Exception {
         assertThat(consumerArn, equalTo(CONSUMER_ARN));
         assertThat(endTime - startTime, greaterThanOrEqualTo(2 * BACKOFF_MILLIS));
 
-        verify(client).registerStreamConsumer(eq(createRegisterStreamConsumerRequest()));
-        verify(client).describeStreamSummary(eq(createDescribeStreamSummaryRequest()));
-        verify(client).describeStreamConsumer(eq(createDescribeStreamConsumerRequest(null)));
-        verify(client, times(2))
-                .describeStreamConsumer(eq(createDescribeStreamConsumerRequest(CONSUMER_ARN)));
+        verify(client).registerStreamConsumer(any(RegisterStreamConsumerRequest.class));
+        verify(client).describeStreamSummary(any(DescribeStreamSummaryRequest.class));
+        verify(client, times(3))
+                .describeStreamConsumer(any(DescribeStreamConsumerRequest.class));
```

Comment:
Same note on this test as above tests, but even more-so now that we have to broaden the search which is picking up an additional request that we didn't care about before. Do we really care if we are calling this API here, especially when considering that we previously ignored one of these invocations?